Resources/Locale/en-US/communications/terror.ftl

```diff
@@ -1,3 +1,3 @@
 terror-dragon = Attention crew, it appears that someone on your station has made an unexpected communication with a strange fish in nearby space.
 terror-revenant = Attention crew, it appears that someone on your station has made an unexpected communication with an otherworldly energy in nearby space.
-terror-loneop = Attention crew, it appears that someone on your station has made an unexpected communication with an unidentified shuttle in nearby space.
+terror-loneop = Attention crew, it appears that someone on your station has made an unexpected communication with an unidentified shuttle in nearby space. # DeltaV - Message for when ninjas summon loneops
```

Comment:
Remove the comment form the same line, place it on the line before it. Else it will display with the message.

Reply:
Scratch that, move it to a DeltaV folder

Reply:
> Scratch that, move it to a DeltaV folder

nuh uh cant make me